### PR TITLE
fix wrong directory name for openssl setup

### PR DESCRIPTION
**Pull request by itsankitkp**
Fixes https://github.com/RustPython/RustPython/issues/4915 by setting correct dir path

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -263,8 +263,8 @@ jobs:
       - name: Set up the Windows environment
         shell: bash
         run: |
-          choco install llvm openssl
-          echo "OPENSSL_DIR=C:\Program Files\OpenSSL-Win64" >>$GITHUB_ENV
+          choco install llvm openssl --no-progress
```
[the remark this answers is not in the document]
Comment (itsankitkp) [reply .github/workflows/ci.yaml:266]: Sorry I don't get it. Both look same to me (or are you talking about extra `--no-progress` that is added in 266)

[the remark this answers is not in the document]
Comment (fanninpm) [reply .github/workflows/ci.yaml:266]: Ah. It seems @youknowone forgot to add _this_ particular one in #4916.

[the remark this answers is not in the document]
Comment (itsankitkp) [reply .github/workflows/ci.yaml:266]: > Ah. It seems @youknowone forgot to add _this_ particular one in #4916.

Yes, I actually ~~ripped off~~ copied it from his PR

[the remark this answers is not in the document]
Comment (youknowone) [reply .github/workflows/ci.yaml:266]: Thank you for catching it 😂

**Review by youknowone — APPROVED** [review]
Thank you so much! I was so much confused about it

**Pull request merged**